backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -476,3 +476,17 @@ databaseChangeLog:
                     foreignKeyName: fk__test_order__test_event
                     references: test_event
                     uniqueKeyName: uk__test_order__test_event
+  - changeSet:
+      id: add-time-to-date-tested
+      author: qot3@cdc.gov
```

Comment:
Nitpick: would use the human-friendly email here rather than the autogenerated one, just to save people looking at `git blame`. 🙂

Reply:
I don't know my human-friendly one